src/strategy/indicators/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from './ema'
 export * from './rsi'
-export * from '/.cmf'
\ No newline at end of file
+export * from '/.cmf'
```

Comment:
2 points here. i think `./cmf` is right here and you removed `export * from './sma'` in your first commit, i think this was not   intended